### DISCUSSION TITLE
Show rule does not work for special characters

### DISCUSSION THREAD

**Discussion by dnlsch** [typst/typst#3935] [Q&A]
I wonder why I can color the text "es" red in the following example, but not the special characters "->"?

Input:
```
#show "es": set text(red) // is working
#show "->": set text(red) // is not working

-> Test
```

Output:
![grafik](https://github.com/typst/typst/assets/118688572/8d753a48-a665-4d71-92db-873db969caac)

**Comment by Enivex** [comment]
See https://github.com/typst/typst/issues/86
